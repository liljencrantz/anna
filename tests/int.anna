
def main(){
    print(1+1,"\n");
    print(2*3,"\n");
    print(1/2,"\n");
    print(5/2,"\n");
    print(4 + 3*3,"\n");
    print(3*3 + 4,"\n");
    print((4+3)*3,"\n");
    print(3*(3+4),"\n");
}
