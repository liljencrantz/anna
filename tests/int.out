2
6
0
2
13
13
21
21
